app/_src/mesh/features/signed-images.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Verify Signatures for Signed Kong Mesh Images
+badge: enterprise
+---
+
+Starting with {{site.mesh_product_name}} 2.7.4, Docker container images are now signed using `cosign` with signatures published to a Docker Hub repository.
+
+This guide provides steps to verify signatures for signed {{site.mesh_product_name}} Docker container images in two different ways:
+
+* A minimal example, used to verify an image without leveraging any annotations
+* A complete example, leveraging optional annotations for increased trust
+
+For the minimal example, you only need Docker image details, a GitHub repo name, and a GitHub workflow filename.
+
+For the complete example, you need the same details as the minimal example, as well as any of the optional annotations you wish to verify:
+
+| Shorthand | Description | Example Value |
+|---|---|---|
+| `<repo>` | Github repository | `kong-mesh` |
+| `<workflow filename>` | Github workflow filename | `kuma-_build_publish.yaml` |
+| `<workflow name>` | Github workflow name | `build-test-distribute` |
+
+Because Kong uses Github Actions to build and release, Kong also uses Github's OIDC identity to sign images, which is why many of these details are Github-related.
+
+## Examples
+
+### Prerequisites
+
+For both examples, you need to:
+
+1. Ensure `cosign` is installed.
+
+2. Collect the necessary image details.
+
+3. Set the `COSIGN_REPOSITORY` environment variable:
+
+   ```sh
+   export COSIGN_REPOSITORY=kong/notary
+   ```
+
+{:.important .no-icon}
+> Github owner is case-sensitive (`Kong/kong-mesh` vs `kong/kong-mesh`).
+
+### Minimal example
```

Comment:
Same here I don't see the point of complete vs minimal. Let's just go complete always it's simpler for users.
Also I would just give something users can copy/paste so no need to have a templated example first and then the full example